circle.yml

```diff
@@ -2,7 +2,7 @@ general:
   branches:
     ignore:
       # We only want to build pull requests for testing. If something is merged,
-      # then we are prepping for release an there is no need to build it again.
+      # then we are prepping for release and there is no need to build it again.
```

Comment:
Normally I'd prefer changes like these come from a PR without recipes. However, this is so minor it seems ok. Though TBH we probably should rewrite this comment as it is a bit long and vague.